tests/test_celery.py

```diff
@@ -1,20 +1,17 @@
 """Unit test module for portal views"""
-import sys
-
-import pytest
+from __future__ import unicode_literals  # isort:skip
 
 from tests import TestCase
 
-if sys.version_info.major > 2:
-    pytest.skip(msg="not yet ported to python3", allow_module_level=True)
 class TestCelery(TestCase):
     """Portal view tests"""
 
     def test_celery_add(self):
         """Try simply add task handed off to celery"""
         x = 151
         y = 99
-        rv = self.client.get('/celery-test?x={x}&y={y}&redirect-to-result=True'. \
-                             format(x=x, y=y), follow_redirects=True)
-        self.assert200(rv)
-        self.assertEqual(rv.data, str(x + y))
+        response = (self.client.get(
+            '/celery-test?x={x}&y={y}&redirect-to-result=True'.
+            format(x=x, y=y), follow_redirects=True))
```

Comment:
Going forward , can you refactor URLs to use `requests.compat.urlencode` (if you're changing a line  for another issue already):
```
response = self.client.get(
    ''.join(
        '/celery-test?',
        requests.compat.urlencode(
            'redirect-to-result': 'True',
            'x': x,
            'y': y,
        ),
    )
)

```

Reply:
Ok, got it